plugins/unifying/lu-device.c

```diff
@@ -866,7 +894,6 @@ lu_device_open (LuDevice *device, GError **error)
 		devid = g_strdup_printf ("USB\\VID_%04X&PID_%04X",
 					 g_usb_device_get_vid (priv->usb_device),
 					 g_usb_device_get_pid (priv->usb_device));
-		lu_device_add_guid (device, devid);
```

Comment:
we still want this surely? we can't take guids away from release to release, we can only add them

Reply:
Indeed. But the guid used in the LVFS currently are the bootloader vid/pid. So this one is not present on the store.